client/publish-icons.py

```diff
@@ -23,18 +23,24 @@ def main(ipfs_local, icon):
 
     with tempfile.TemporaryDirectory() as tmp:
```

Comment:
Sorry for the pre-emptive review request yesterday, I assumed that there would be only trivial changes, after yesterday. But I'd like you to review/note the changes in the ipfs stuff here.